### PR TITLE
present how to build the documentation faster

### PR DESCRIPTION
**Pull request by tudorbarascu**

### CODE REVIEW

**Comment by tudorbarascu** [comment]
@dassau Based on you recent mailing list inquiries I think this may be useful to you.

File: readme.rst [hunk 1]
```diff
@@ -106,6 +106,27 @@ Then build:
 * ``make html`` to build the english language
 * ``make LANG=nl html`` to build the dutch version
 
+**Tip**
```
Comment (yjacolin) [inline readme.rst]: we can use `.. note::` here ?

Comment (DelazJ) [inline readme.rst]: Given the length of the comment, why not create a subsection with a title? I think that notes and tips should not be too long.

**Comment by tudorbarascu** [comment]
Just turned into a subsection.
Actually, last time I checked ``.. note::`` or other tags are NOT interpreted by the github markup parser (the readme file doesn't build with sphinx-docs) so they're out of question.
Think it's good now.

**Comment by tudorbarascu** [comment]
Should be ready to go

**Comment by DelazJ** [comment]
same thing for `:file:`?

**Comment by tudorbarascu** [comment]
@DelazJ same.. same. I was too much into the documentation guidelines :). Thanks for assisting. It's good to go. Again.

**Pull request merged**